.github/workflows/tests.yml

```diff
@@ -0,0 +1,54 @@
+name: iceberg-tests
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  iceberg-isolation-test:
+    name: Test lakeFS isolation with iceberg
+    runs-on: ubuntu-20.04
+    services:
+      lakefs:
+        image: "treeverse/lakefs:latest"
```

Comment:
May be better to lock the version number here. There is a trade-off, but I think consistency wins over freshness.